internal/ignition/templates/discovery.ign

```diff
@@ -191,6 +191,14 @@
         "name": "root"
       },
       "contents": { "source": "data:text/plain;base64,{{.OKDHoldAgent}}" }
+    }{{end}}{{if .PROXY_SETTINGS}},
+    {
+      "path": "/etc/profile.d/proxy.sh",
+      "mode": 420,
```

Comment:
All other files under `/etc/profile.d/*.sh` are `-rw-r--r--` (`644`) so why `420`?

Reply:
Good point, I just changed it.